src/main/kotlin/org/hyacinthbots/lilybot/extensions/util/Tags.kt

```diff
@@ -149,6 +150,34 @@ class Tags : Extension() {
 							"${arguments.user?.mention ?: ""}\n**${tagFromDatabase.tagTitle}**\n${tagFromDatabase.tagValue}"
 					}
 				}
+
+				// Log when a message tag is sent to allow identification of tag spammers
+				if (tagFromDatabase.tagAppearance == "message") {
+					getLoggingChannelWithPerms(
+						guild!!.asGuild(),
+						UtilityConfigCollection().getConfig(guild!!.id)!!.utilityLogChannel ?: guild!!.asGuild()
```

Comment:
We definitely don't want this going to a guild's system channel. I would be very annoyed by that. Just don't log it if there's no utility log set.